src/routes/albums.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const albumController = require('../controllers/albums.js');
+const router = express.Router();
+
+router.get('/', albumController.retrieveAlbums);
+router.get('/:id', albumController.getAlbumById);

```

Comment:
Here you used the name getAlbumById, I would continue use the same naming when the id is used, like updateAlbumById deleteAlbumById...